package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "acl-sequelize",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
This has to be a major change. But any way leave the version as is for now

Reply:
You mean leave 0.1.2?